package.json

```diff
@@ -149,8 +149,8 @@
 		"resize-observer-polyfill": "1.5.1",
 		"rimraf": "3.0.2",
 		"rtlcss": "4.0.0",
-		"sass": "1.35.2",
-		"sass-loader": "12.1.0",
+		"sass": "1.50.1",
```

Comment:
There is a more up-to-date version of `sass,`, but there are too many deprecations reported in the Gutenberg codebase, so I went with the highest possible version that doesn't warn. It isn't essential for the fix, but I thought it would be great to bump the version as part of the effort.